src/AppBundle/Utils/Slugger.php

```diff
@@ -27,6 +27,6 @@ class Slugger
      */
     public function slugify($string)
     {
-        return trim(preg_replace('/[^a-z0-9]+/', '-', strtolower(strip_tags($string))), '-');
+        return str_replace(' ', '-', strtolower(trim(strip_tags($string))));
```

Comment:
you should replace any whitespace char, not only spaces. And you should not put several dashes for consecutive spaces.